source/docs/networking/measuring-bandwidth-usage.rst

```diff
@@ -0,0 +1,88 @@
+Measuring Bandwidth Usage
+=========================
+
+On the 2013 FRC Field (and at home when the DAP-1522 is configured using the FRC Bridge Configuration Utility) each team is limited to 7Mb/s of network traffic (see the `FMS Whitepaper <http://www.usfirst.org/sites/default/files/uploadedFiles/Robotics_Programs/FRC/Game_and_Season__Info/2013/FMSWhitePaper_RevA.pdf>`__ for more details). The FMS Whitepaper provides information on determining the bandwidth usage of the Axis camera, but some teams may wish to measure their overall bandwidth consumption. This document details how to make that measurement.
```

Comment:
We should fix the link.  I think the current version is hosted on ScreenSteps.

Reply:
Do you think the [FMS whitepaper page on screensteps](https://wpilib.screenstepslive.com/s/currentCS/m/troubleshooting/l/705152-fms-whitepaper) should be ported or is there another PDF that should be linked?

Reply:
That page is just an alias for this page: https://wpilib.screenstepslive.com/s/fms/m/whitepaper/l/608744-fms-whitepaper

We should link there for now, and may be deciding to port that FMS site as well. If we do, we'll update the link then.